index.html

```diff
@@ -39,11 +43,135 @@ <h1>Nsong Fitz Gerald</h1>
                 <div class="button">
                     <button id="introbutton">Start collaboration</button>
                 </div>
+            </div> 
+        </section>
+        <section>
+            <div id="titleimg"></div>
+            <div id="projecttitle">
+                <h1 class="title">Projects</h1>
             </div>
-
         </section>
-       
-
+    <section id="projects">
+        <div class="project">
+            <div class="projectimg"><img src="./assets/img/project1.png" alt="project1"></div>
+            <div class="projectdetails">
+                <h3>Project name goes here</h3>
+                <div class="projectbuttons">
+                    <button>HTML/CSS</button>
+                    <button>Ruby on Rails</button>
+                    <button>JavaScript</button>
+                </div>
+                <div class="viewbutton">
+                    <p>See this project</p>
+                    <i class="fa-solid fa-arrow-right"></i>
+                </div>
+            </div>
+        </div>
+        <div class="project">
+            <div class="projectimg"><img src="./assets/img/project2.png" alt="project1"></div>
+            <div class="projectdetails">
+                <h3>Project name goes here</h3>
+                <div class="projectbuttons">
+                    <button>HTML/CSS</button>
+                    <button>Ruby on Rails</button>
+                    <button>JavaScript</button>
+                </div>
+                <div class="viewbutton">
+                    <p>See this project</p>
+                    <i class="fa-solid fa-arrow-right"></i>
+                </div>
+            </div>
+        </div>
+        <div class="project">
+            <div class="projectimg"><img src="./assets/img/project3.png" alt="project1"></div>
+            <div class="projectdetails">
+                <h3>Project name goes here</h3>
+                <div class="projectbuttons">
+                    <button>HTML/CSS</button>
+                    <button>Ruby on Rails</button>
+                    <button>JavaScript</button>
+                </div>
```

Comment:
- [ ] For semantic reasons, kindly use `ul` tags and `li` tags to list the technology used in each card/project. You can see the example in the first reviewer's comment. https://github.com/fitz95/Portfolio-Microverse-/pull/3/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&file-filters%5B%5D=.md&show-viewed-files=true#r1038761165